docs/auditors/capabilities.md

```diff
@@ -49,81 +31,151 @@ $ kubeaudit capabilities -f "auditors/capabilities/fixtures/capabilities-nil.yml
 
 --------------------------------------------
 
--- [error] CapabilityNotDropped
-   Message: Capability not dropped. Ideally, the capability drop list should include the single capability 'ALL' which drops all capabilities.
-   Metadata:
-      Container: container
-      Capability: AUDIT_WRITE
-
--- [error] CapabilityNotDropped
-   Message: Capability not dropped. Ideally, the capability drop list should include the single capability 'ALL' which drops all capabilities.
+-- [error] CapabilityOrSecurityContextMissing
+   Message: Security Context not set. Ideally, the Security Context should be specified and all Capabilities should be dropped by setting the Drop list to ALL.
    Metadata:
       Container: container
-      Capability: CHOWN
 
-...
 ```
 
-### Example with Custom Drop List
+### Example with Config File
+
+A custom add list can be provided in the config file. See [docs](docs/all.md) for more information. These are the capabilities you'd like to add and not have kubeaudit raise an error. In this example, kubeaudit will only error for "CHOWN" because it wasn't added to the add list in the config.
 
-A custom drop list can be provided as a space-separated list of capabilities using the `-d/--drop` flag:
+`example.yaml` (config)
 
+```yaml
+---
+auditors:
+  capabilities:
+    # add capabilities needed to the add list, so kubeaudit won't report errors
+    add: ['KILL', 'MKNOD']
+```
+
+`test.yaml`(manifest)
+
+```yaml
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: deployment
+  namespace: capabilities-some-allowed-multi-containers-some-labels
+spec:
+  selector:
+    matchLabels:
+      name: deployment
+  template:
+    metadata:
+      labels:
+        name: deployment
+    spec:
+      containers:
+        - name: container1
+          image: scratch
+          securityContext:
+            capabilities:
+              add:
+                - CHOWN
+                - KILL
+                - MKNOD
+              drop:
+                - ALL
 ```
-$ kubeaudit capabilities --drop "MAC_ADMIN AUDIT_WRITE" -f "auditors/capabilities/fixtures/capabilities-nil.yml"
+
+```shell
+$ kubeaudit all --kconfig "example.yaml" -f "test.yaml"
```

Comment:
Can we name the two files more descriptively so this command is easier to understand? eg `config.yaml` and `manifest.yaml`